cloudbuild.yaml

```diff
@@ -4,7 +4,7 @@ options:
   # job builds a multi-arch docker image for amd64,arm,arm64 and s390x.
   machineType: E2_HIGHCPU_32
 steps:
-  - name: "gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20221007-ad65926f6b"
+  - name: "gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20230424-910a2a439d"
```

Comment:
is this bump mandatory?

Reply:
https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/post-kwok-push-images/1654674164432244736

The latest PR got merged, but it caused a build failure due to using a new feature in Go 1.20, just upgraded the Go version of the release build.